draft-ietf-quic-load-balancers.md

```diff
@@ -312,7 +293,10 @@ the servers, but the algorithm SHOULD be deterministic over short time scales so
 that related packets go to the same server.
 
 A fallback algorithm MAY parse a packet from a QUIC version it understands and
-use that information to make a routing decision.
+use that information to make a routing decision. If so, it MAY buffer packets
+with unroutable DCIDs to await further packets that allow it to make a routing
+decision. For example, it may use the TLS 1.3 Server Name Indication (SNI)
+field, which might appear in one of several QUIC version 1 Initial packets.
 
```

Comment:
What about Post-Quantum? Do you mean, buffer the initial packets until the SNI can be retrieved from the Client Hello contained in the Crypto frames"? What if the SNI straddles multiple QUIC packets? What if ECH is used?

Reply:
I think that's what this is saying, right? "SNI field, which might appear in one of several... packets". Is there another wording you would suggest/

As for ECH, it seems clear to me that the LB would have to be the "client-facing server" in Fig. 2 of the ESNI draft. I don't know that this draft has to get into the details of how to implement arbitrary routing strategies. WDYT?

Reply:
OK.